apps/chordstop/components/ArtistSocialMedia.tsx

```diff
@@ -1,4 +1,4 @@
-import { Artist } from "@prisma/client";
+import { Artist } from ".prisma/client";
```

Comment:
it should be @prisma/client... if it doesn't work than it means something has not been set up quite right

Reply:
I changed it check the prisma schema so it doesn't conflict

Reply:
not sure what you mean, let's discuss... i know in some forums they say to use .prisma but I know for first-hand-exeprience it shouldn't be so